src/fragments/lib/logging/common/remote_configuration/remote_configuration.mdx

```diff
@@ -0,0 +1,220 @@
+    const lambdaConfig = 'resources/lambda/remoteconfig.js'
+    const configFileName = 'remoteloggingconstraints.json'
+    const logGroupName = <CloudWatch-log-group-name>
+    const authRoleName = <Amplify-authenticated-role-name> 
+    const unAuthRoleName = <Amplify-unauthenticated-role-name>
+    const bucketName = <S3-bucket-name>
+
+    new logs.LogGroup(this, 'Log Group', {
+      logGroupName: logGroupName,
+      retention: logs.RetentionDays.INFINITE
+    })
+    
+    const authRole = iam.Role.fromRoleName(this, "Auth-Role", authRoleName)
+    const unAuthRole = iam.Role.fromRoleName(this, "UnAuth-Role", unAuthRoleName)
+    const logResource = `arn:aws:logs:${region}:${account}:log-group:${logGroupName}:log-stream:*`
+    const logIAMPolicy = new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      resources: [logResource],
+      actions: ["logs:PutLogEvents", "logs:DescribeLogStreams", "logs:CreateLogStream"]
+    })
+
+    authRole.addToPrincipalPolicy(logIAMPolicy)
+    unAuthRole.addToPrincipalPolicy(logIAMPolicy)
+
+    const remoteConfigBucket = new s3.Bucket(this, 'AmplifyRemoteLogging-Bucket', {
+        publicReadAccess: false,
+        versioned: true,
+        bucketName: bucketName
+      });
+
+    new BucketDeployment(this, `AmplifyRemoteLogging-BucketDeployment`, {
+        sources: [
+            Source.asset(path.dirname(path.join(loggingConfigLocation))),
+        ],
+        destinationBucket: remoteConfigBucket
+    });
+
+    const handler = new lambda.Function(this, "AmplifyRemoteLogging-Handler", {
+      runtime: lambda.Runtime.NODEJS_18_X,
+      code: lambda.Code.fromAsset(path.dirname(path.join(lambdaConfig))),
+      handler: "remotelogging.main",
+      environment: {
+        BUCKET: remoteConfigBucket.bucketName,
+        KEY: configFileName
+      }
+    })
+
+    remoteConfigBucket.grantRead(handler)
+
+    const api = new apigateway.RestApi(this, "AmplifyRemoteLogging-API", {
+      restApiName: "Logging API",
+      description: "API Gateway for Remote Logging"
+    })
+
+    const getRemoteLoggingIntegration = new apigateway.LambdaIntegration(handler)
+    const loggingConstraints = api.root.addResource('loggingconstraints')
+    const getLoggingConstraints = loggingConstraints.addMethod('GET', getRemoteLoggingIntegration, {
+      authorizationType: apigateway.AuthorizationType.IAM
+    })
+
+    const apiInvokePolicy = new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      resources: [ getLoggingConstraints.methodArn ],
+      actions: ['execute-api:Invoke']
+    })
+
+    authRole.addToPrincipalPolicy(apiInvokePolicy)
+    unAuthRole.addToPrincipalPolicy(apiInvokePolicy)
+
+  }
+}
+```
+
+## Sample Lambda handler
+Below is a sample lambda that reads and returns the `remoteloggingconstraints.json` from AWS S3.
+Note that caching and ETag is used.  To reduce data usage, if the logging levels has not changed since the last fetch, there is no need to return the log levels.
```

Comment:
To reduce data usage, if the logging levels has not changed since the last fetch, there is no need to return the log levels.

This reads a bit awkwardly to me - what does this mean?

Reply:
The intent is to talk about caching to reduce bandwidth usage.  I've made an attempt to re-word, let me know if it's still confusing or better to remove the note completely.